cpp/src/Slice/Parser.cpp

```diff
@@ -64,10 +64,12 @@ Slice::CompilerException::reason() const
     return _reason;
 }
 
+// Forward declare things from Bison and Flex the parser can use.
+extern int slice_parse();
+extern int slice_lineno;
 extern FILE* slice_in;
 extern int slice_debug;
-
-int slice_parse();
+extern int slice__flex_debug;
```

Comment:
is the double `__` correct?

Reply:
yes, unfortunately flex can't make up it's mind between using `yy...` and `yy_...` as variable prefixes, and we set a macro that replaces `yy` with `slice_`, so in some instances we end up with this double underscore.